flatten-array.md

```diff
@@ -1,8 +1,11 @@
-The challenge is to write a function that accepts an arbitrarily-deep nested list-like structure and returns a flattened structure without any null/nill values. 
- 
-For Example
+The challenge is to write a function that accepts an arbitrarily-deep nested list-like structure and returns a flattened structure without any null/nill values.
+
+For example:
 
 input: [1,[2,3,null,4],[null],5]
 
 output: [1,2,3,4,5]
 
+## Note
+
+Implement the function yourself. Do not use something else to perform the flattening for you.
```

Comment:
Is this true for all the language tracks?

I know that in some tracks it's useful to help people get to know the core libraries.

I guess the question here, @exercism/track-maintainers, is should this be a general guide, or should this be placed in a `HINTS.md` file in this exercise for particular languages.